### PR TITLE
Add Dust Racing 2D

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/juzzlin/DustRacing2D/issues/124
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. https://github.com/juzzlin/DustRacing2D/pull/125

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.github.juzzlin.DustRacing2D

**Comment by Mailaender** [comment]
bot, build com.github.juzzlin.DustRacing2D

**Comment by Mailaender** [comment]
bot, build com.github.juzzlin.DustRacing2D

File: com.github.juzzlin.DustRacing2D.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+rename-icon: dustrac-game
+rename-desktop-file: dustrac-game.desktop
+rename-appdata-file: dustrac.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=home
```
Comment (barthalion) [inline com.github.juzzlin.DustRacing2D.yml:15]: Why does it need unrestricted home access?

**Comment by hfiguiere** [comment]
bot, build com.github.juzzlin.DustRacing2D

File: com.github.juzzlin.DustRacing2D.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+rename-icon: dustrac-game
+rename-desktop-file: dustrac-game.desktop
+rename-appdata-file: dustrac.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-config/.dustrac
```
Comment (barthalion) [inline com.github.juzzlin.DustRacing2D.yml:15]:
```suggestion
```

This is not needed either, the app respects XDG_CONFIG_HOME.

Comment (Mailaender) [reply com.github.juzzlin.DustRacing2D.yml]: Removed it.

**Pull request closed without merging**